CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# Линии, начинающиеся с '#' — это комментарии.
+# Каждая линия — это шаблон файлов, сопровождаемый одним
+# или несколькими участниками.
+
+# Следующие участники будут выбраны ревьюерами по-умолчанию
+# для всех файлов репозитория.
```

Comment:
That's OK, but could you please make sure that at least one mentor's approval is required and 3 approvals in total are required?